cmd/attack/network.go

```diff
@@ -114,6 +114,7 @@ func NewNetworkLossCommand(dep fx.Option, options *core.NetworkCommand) *cobra.C
 	cmd.Flags().StringVarP(&options.Hostname, "hostname", "H", "", "only impact traffic to these hostnames")
 	cmd.Flags().StringVarP(&options.IPProtocol, "protocol", "p", "",
 		"only impact traffic using this IP protocol, supported: tcp, udp, icmp, all")
+	cmd.Flags().BoolVar(&options.FullDisable, "full-disable", false, "full net on device disable flag")
```

Comment:
This flag is also useful for other network attacks. Can you help to add this flag for other network attacks?

Reply:
Yes, of course, I will supplement my commit with this in the near future

Reply:
@cwen0 , I have added this flag for attacks that can implement it. Review, please.